client/me/account-close/closed.jsx

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import EmptyContent from 'components/empty-content';
+import Spinner from 'components/spinner';
+import getPreviousRoute from 'state/selectors/get-previous-route';
+import isAccountClosed from 'state/selectors/is-account-closed';
+
+/**
+ * Style dependencies
+ */
+import './closed.scss';
+
+class AccountSettingsClosedComponent extends Component {
+	render() {
+		/* eslint-disable-next-line no-shadow */
+		const { isAccountClosed, previousRoute, translate } = this.props;
+
+		if ( previousRoute !== '/me/account/close' ) {
+			page( '/me/account/close' );
+		}
+
+		if ( ! isAccountClosed ) {
+			return (
+				<div className="account-close__spinner">
+					<Spinner size={ 32 } />
+					<p className="account-close__spinner-text">
+						{ translate( 'Your account is being deleted' ) }
+					</p>
+				</div>
+			);
+		}
+
+		return (
+			<EmptyContent
+				title={ translate( 'Your account has been closed' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'User Account has been closed.' )` *ES Score: 7*